packages/sanity/src/core/releases/tool/detail/documentTable/DocumentRow.tsx

```diff
@@ -0,0 +1,225 @@
+import {CheckmarkCircleIcon, EmptyIcon, Progress50Icon} from '@sanity/icons'
+import {type SanityDocument} from '@sanity/types'
+import {AvatarStack, Box, Card, Flex, Text} from '@sanity/ui'
+import {
+  type Dispatch,
+  type ForwardedRef,
+  forwardRef,
+  type SetStateAction,
+  useEffect,
+  useMemo,
+} from 'react'
+import {useObservable} from 'react-rx'
+import {
+  getPreviewStateObservable,
+  getPreviewValueWithFallback,
+  getPublishedId,
+  RelativeTime,
+  SanityDefaultPreview,
+  type SchemaType,
+  useDocumentPreviewStore,
+  UserAvatar,
+  useSchema,
+} from 'sanity'
+import {IntentLink} from 'sanity/router'
+
+import {Tooltip} from '../../../../../ui-components'
+import {type BundleDocument} from '../../../../store/bundles/types'
+import {useVersionHistory} from './useVersionHistory'
+
+const DOCUMENT_STATUS = {
+  ready: {
+    icon: CheckmarkCircleIcon,
+    color: 'var(--card-badge-positive-icon-color)',
+    text: 'This document is marked as ready',
+  },
+  noChanges: {
+    icon: EmptyIcon,
+    color: '',
+    text: "This document doesn't have changes in this release",
+  },
+  edited: {
+    icon: Progress50Icon,
+    color: 'var(--card-badge-caution-icon-color)',
+    text: 'This document has been edited in this release',
+  },
+} as const
```

Comment:
All of this needs confirmation, will be addressed in future changes once Marius is back, this are just placeholders text and icons